ChromeOs specific patch here - *ALL* chromeos arm platforms are
gnueabi with hardfp, in chromeos toolchain, we call such configuration
"arm-linux-gnueabi", notwithstanding, clang calls this
"arm-linux-gnueabihf". It might be better to change our toolchain
configuration, but that would be much more complex.
diff --git a/lib/Support/Triple.cpp b/lib/Support/Triple.cpp
index b862dbd..4e93670 100644
--- a/lib/Support/Triple.cpp
+++ b/lib/Support/Triple.cpp
@@ -378,11 +378,16 @@ static Triple::OSType parseOS(StringRef OSName) {
 }
 
 static Triple::EnvironmentType parseEnvironment(StringRef EnvironmentName) {
+  // ChromeOs specific patch here - *ALL* chromeos arm platforms are gnueabi
+  // with hardfp, in chromeos toolchain, we call such configuration
+  // "arm-linux-gnueabi", notwithstanding, clang calls this
+  // "arm-linux-gnueabihf". It might be better to change our toolchain
+  // configuration, but that would be much more complex.
   return StringSwitch<Triple::EnvironmentType>(EnvironmentName)
     .StartsWith("eabihf", Triple::EABIHF)
     .StartsWith("eabi", Triple::EABI)
     .StartsWith("gnueabihf", Triple::GNUEABIHF)
-    .StartsWith("gnueabi", Triple::GNUEABI)
+    .StartsWith("gnueabi", Triple::GNUEABIHF)
     .StartsWith("gnux32", Triple::GNUX32)
     .StartsWith("code16", Triple::CODE16)
     .StartsWith("gnu", Triple::GNU)
